pkg/cli/init.go

```diff
@@ -50,9 +58,17 @@ Upstream can be one of:
 	cmd.Flags().Bool("rm-asset-dest", false, "Always remove asset destinations if already present")
 	cmd.Flags().Int("retries", 3, "Number of times to retry retrieving upstream")
 
-	viper.BindPFlags(cmd.Flags())
-	viper.BindPFlags(cmd.PersistentFlags())
-	viper.AutomaticEnv()
-	viper.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))
+	// optional flags for "ship init replicated.app"
+	cmd.Flags().String("customer-id", "", "Customer ID for which to query app specs. Required for all ship operations.")
+	cmd.Flags().String("installation-id", "", "Installation ID for which to query app specs")
+	cmd.Flags().String("release-semver", "", "Specific release version to pin installation to. Requires channel-id")
+
+	// optional developer flags for "ship init replicated.app"
+	cmd.Flags().String("runbook", "", developerFlagUsage)
```

Comment:
this one should go away, the `runbook` flag isn't necessary since we specify the `ship.yaml` to use via the argument to `init`